website/docs/r/cwf_process.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: CWF Process"
+sidebar_current: "docs-rightscale-resource-cwf_process"
+description: |-
+  Create and maintain a rightscale CloudWorkFlow process.
+---
+
+# rightscale_cwf_process
+
+Use this resource to create or destroy RightScale [CloudWorkFlow processes](http://docs.rightscale.com/ss/reference/rcl/).
+
+Creating the CWF process will run it synchronously and return the output values (if any).
+
+Destroying a CWF process will remove it from the CWF Console table of executed processes. It WON'T undo any action that it could have executed nor would stop any running process.
```

Comment:
The fact that processes are listed in the CWF Console is ancillary really. The description should just specify that the process gets deleted. Also deleting a process does cause it to stop (in error).